routes/articles.js

```diff
@@ -0,0 +1,126 @@
+const express = require('express');
+const router = express.Router();
+
+const Articles = require('../db/articles');
+const articlesDB = new Articles();
+
+// Routes /articles to index and sends an object with articles object
+
+router.route('/')
+  .get((req, res) => {
+    res.render('index', { 
+      articles : {
+          showArticles : true,
+          showFunction : articlesDB.showAll()
+      }
+    });
+  });
+
+router.route('/new')
+  .get((req, res) => {
+    res.render('index', { 
+      articles : {
+        new : true
```

Comment:
A more appropriate variable name could be `isNew` this tells the reader that it's a boolean.